Makefile.j2

```diff
@@ -25,20 +25,20 @@ generate-ontologies: $(ONTOLOGYDIR)/{{ ontology.id }}.owl
 mirror-{{ ontology.id }}:
 	mkdir -p $(TMP_DATA)
 	if [ $(MIR) = true ]; then curl -L {{ ontology.mirror_from }} --create-dirs -o $(ONTOLOGYDIR)/{{ ontology.id }}-download.owl  --max-time 600 &&\
-		$(ROBOT) convert -i $(ONTOLOGYDIR)/{{ ontology.id }}-download.owl -o $@.tmp.owl && mv $@.tmp.owl $(TMP_DATA)/$@.owl; fi
+		$(ROBOT) merge -i $(ONTOLOGYDIR)/{{ ontology.id }}-download.owl convert -o $@.tmp.owl && mv $@.tmp.owl $(TMP_DATA)/$@.owl; fi
 
 {% if 'parent_term' in ontology -%}
 
 
 $(TMP_DATA)/{{ ontology.id }}_terms.txt: $(ONTOLOGYDIR)/{{ ontology.id }}.owl
-	runoak -i sqlite:$< descendants -p i {{ ontology.parent_term }} > $@
+	runoak -i sqlite:$< descendants -p i {{ ontology.parent_term }} -o $@
 
 {% endif %}
 
 generate-ontologies: $(ONTOLOGYDIR)/{{ ontology.id }}.db
 
 $(ONTOLOGYDIR)/{{ ontology.id }}.db: $(ONTOLOGYDIR)/{{ ontology.id }}.owl
-	semsql make $@
+	./odk.sh semsql make $@
```

Comment:
Which was first, the hen or the egg? The question this raises is if wrap the ODK inside the makefile or run the makefile inside the ODK. I prefer the latter. So rather than `make generate-ontologies` you run `sh run.sh make generate-ontologies`, which executes _all_ the pipeline inside the ODK.

```suggestion
	semsql make $@
```

This edit will have to be made everywhere in the PR.